pgml-extension/examples/binary_classification.sql

```diff
@@ -1,5 +1,5 @@
 -- Exit on error (psql)
-\set ON_ERROR_STOP true
+-- \set ON_ERROR_STOP true
```

Comment:
I think this is going to cause a problem later when we miss breakage. These are intended to be part of CI.

Reply:
https://github.com/postgresml/postgresml/blob/master/pgml-extension/tests/test.sql#L9